### PR TITLE
feat(container): replace cookies policy modal on odsmodal

### PR DESCRIPTION
**Pull request by stif59100**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/MANAGER-12303
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | MANAGER-13842
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~

## Description

I replace react-boostrap modal to ods modal on cookies policies

### CODE REVIEW

**Review by anooparveti — APPROVED** [review]

File: packages/manager/apps/container/src/cookie-policy/CookiePolicy.tsx [hunk 1]
```diff
@@ -1,39 +1,58 @@
...
 import links from './links';
-import './style.scss';
-import modalStyle from './cookie-modal.module.scss';
-
 import { useApplication } from '@/context';
+import { Subtitle, Links, LinksProps } from '@ovhcloud/manager-components';
+import { OsdsButton, OsdsModal, OsdsText} from '@ovhcloud/ods-components/react';
+import { ODS_BUTTON_SIZE, ODS_BUTTON_VARIANT, ODS_TEXT_COLOR_HUE, ODS_TEXT_LEVEL, ODS_TEXT_SIZE } from '@ovhcloud/ods-components';
+import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';
+import { OdsHTMLAnchorElementTarget } from "@ovhcloud/ods-common-core";
```
Comment (chipp972) [inline packages/manager/apps/container/src/cookie-policy/CookiePolicy.tsx:13]:
```suggestion
import { OdsHTMLAnchorElementTarget } from '@ovhcloud/ods-common-core';
```

File: packages/manager/apps/container/src/cookie-policy/CookiePolicy.tsx [hunk 2]
```diff
@@ -1,39 +1,58 @@
...
 
+const ModalContent = ({ label }: { label: string }) => (
+  <OsdsText level={ODS_TEXT_LEVEL.caption} color={ODS_THEME_COLOR_INTENT.text} size={ODS_TEXT_SIZE._300} hue={ODS_TEXT_COLOR_HUE._500}>
+    {label}
+  </OsdsText>
+);
+
 const CookiePolicy = ({ shell }: Props): JSX.Element => {
-  const { t } = useTranslation('cookie-policy');
+   const { t } = useTranslation('cookie-policy');
```
Comment (chipp972) [inline packages/manager/apps/container/src/cookie-policy/CookiePolicy.tsx:27]: can you format the file correctly ?

File: packages/manager/apps/container/src/cookie-policy/CookiePolicy.tsx [hunk 3]
```diff
@@ -1,39 +1,58 @@
...
   const [cookies, setCookies] = useCookies(['MANAGER_TRACKING']);
   const { environment } = useApplication();
   const [show, setShow] = useState(false);
-  const { ovhSubsidiary } = shell.getPlugin('environment').getEnvironment()
-    .user as User;
+  const { ovhSubsidiary } = shell.getPlugin('environment').getEnvironment().user as User;
```
Comment (chipp972) [inline packages/manager/apps/container/src/cookie-policy/CookiePolicy.tsx:31]: Why don't we just use `environment.user.ovhSubsidiary` ?

File: packages/manager/apps/container/src/cookie-policy/CookiePolicy.tsx [hunk 4]
```diff
@@ -49,71 +68,46 @@ const CookiePolicy = ({ shell }: Props): JSX.Element => {
...
-          <h4 className={`${modalStyle.title} text-center`}>
-            {t('cookie_policy_title')}
-          </h4>
-          <p dangerouslySetInnerHTML={moreInfoHtml()}></p>
-          <ul className={modalStyle.list}>
-            <li>{t('cookie_policy_description_3')}</li>
-          </ul>
-          <p className="mb-3">{t('cookie_policy_description_2')}</p>
-          <span>{t('cookie_policy_description_4')}</span>
+        <OsdsButton slot='actions' size={ODS_BUTTON_SIZE.sm} variant={ODS_BUTTON_VARIANT.stroked} color={ODS_THEME_COLOR_INTENT.primary} onClick={deny}>
```
Comment (chipp972) [inline packages/manager/apps/container/src/cookie-policy/CookiePolicy.tsx:102]:
```suggestion
        <OsdsButton slot="actions" size={ODS_BUTTON_SIZE.sm} variant={ODS_BUTTON_VARIANT.stroked} color={ODS_THEME_COLOR_INTENT.primary} onClick={deny}>
```

Comment (chipp972) [inline packages/manager/apps/container/src/cookie-policy/CookiePolicy.tsx:102]:
```suggestion
        <OsdsButton slot="actions" size={ODS_BUTTON_SIZE.sm} variant={ODS_BUTTON_VARIANT.stroked} color={ODS_THEME_COLOR_INTENT.primary} onClick={deny}>
```

File: packages/manager/apps/container/src/cookie-policy/CookiePolicy.tsx [hunk 5]
```diff
@@ -49,71 +68,46 @@ const CookiePolicy = ({ shell }: Props): JSX.Element => {
...
-  const moreInfoLink = links.moreInfo[ovhSubsidiary];
-  const clickHereLink = links.changeOpinionLink[ovhSubsidiary];
+  return (<>
+    {show && (
+      <OsdsModal dismissible onOdsModalClose={() => setShow(false)}>
+      <div className="p-1">
+        <div className="w-full flex justify-center items-center">
+          <img src={ovhCloudLogo} alt="ovh-cloud-logo" />
+        </div>
+        <div className='text-center m-4'>
```
Comment (chipp972) [inline packages/manager/apps/container/src/cookie-policy/CookiePolicy.tsx:80]:
```suggestion
        <div className="text-center m-4">
```

**Review by chipp972 — DISMISSED** [review]

**Review by chipp972 — DISMISSED** [review]

**Comment by chipp972** [comment]
You have to rebase correctly on master for the tests to pass

**Review by JacquesLarique — APPROVED** [review]

**Pull request merged**